doc/tutorial/remote-apis.md

```diff
@@ -0,0 +1,105 @@
+# Remote APIs
+
+JSTP allows you to use remote APIs as regular asynchronous functions, let's
+examine how it looks like by updating our `client.js` file like this:
+
+```javascript
+'use strict';
+
+const jstp = require('@metarhia/jstp');
+
+const APP_NAME = 'chatApp';
+const PORT = 3000;
+const HOSTNAME = '127.0.0.1';
+
+jstp.ws.connectAndInspect(
+  APP_NAME,
+  null,
+  ['auth', 'chat'],
+  {},
+  `ws://${HOSTNAME}:${PORT}`,
+  (error, connection, app) => {
+    if (error) {
+      console.error(error);
+      return;
+    }
+
+    console.log(`Connected to ${APP_NAME}`);
+
+    const username = 'user_1';
+    const password = 'password_1';
+
+    app.auth.register(username, password, error => {
+      if (error) {
+        console.error(error);
+        return;
+      }
+
+      console.log('Registered');
+
+      app.auth.login(username, password, error => {
+        if (error) {
+          console.error(error);
+          return;
+        }
+
+        console.log(`Logged in as ${username}`);
+      });
+    });
+  }
+);
+```
+
+> The code added in callback at this step works both in Node.js and browser, so
```

Comment:
@belochub fix here a problem mentioned in this comment https://github.com/metarhia/jstp/pull/417#discussion_r258745543